pkg/sanity/sanity.go

```diff
@@ -155,6 +155,16 @@ type TestConfig struct {
 	// generator for valid Volume and Node IDs. Defaults to
 	// DefaultIDGenerator.
 	IDGen IDGenerator
+
+	// Repeat count for Volume operations to test idempotency requirements.
+	// csi-test/pkg/sanity/node.go:runControllerTest() optionally runs
+	// added variant set of tests which repeats those Volume operations
+	// that are required to be idempotent, based on this count value.
+	// -1 : Skip idempotency test
+	//  0 : Use default value, 10 repetitions
+	//  n : repeat n times
+	// Note that missing explicit value leads to default repetition count.
+	IdempotentCount int
```

Comment:
It just occurred to me that we no longer need this special logic with "0" meaning "default repetitions" because the default can now be set directly in https://github.com/kubernetes-csi/csi-test/blob/4560adb95d44a5ffeac46ea81fbb108979974e80/pkg/sanity/sanity.go#L188-L200

Reply:
@okartau: this is still open.